src/ngraph/runtime/cpu/builder/embedding_lookup.cpp

```diff
@@ -77,6 +78,19 @@ namespace ngraph
                                 in_shape);
                         };
                     }
+                    else if (index_element_type == element::i64)
+                    {
+                        functor = [&, in_shape, element_count](CPURuntimeContext* ctx,
+                                                               CPUExecutionContext* ectx) {
+
+                            ngraph::runtime::reference::embedding<float, long>(
+                                static_cast<long*>(arg0_tensor),
```

Comment:
Use types like `int64_t` instead of `long` since long is only guaranteed to be at least 32 bits

Reply:
@rkimballn1 : Good catch.  Looks like my assumption was [pretty safe for 64-bit OSX and Linux, but not for Win64](http://nickdesaulniers.github.io/blog/2016/05/30/data-models-and-word-size/).